src/notification.cpp

```diff
@@ -33,6 +33,8 @@
 #include "notification.h"
 #include "notification_p.h"
 
+#include <QVariantHash>
+#include <QDBusArgument>
```

Comment:
Including `QDBusArgument` and `QVariantHash` here may not be needed since it is already include in `notification_p.h`.

Reply:
Sure, I'll just remove then.